packages/ui/package.json

```diff
@@ -96,7 +96,8 @@
     "workbox-expiration": "^6.1.0",
     "workbox-precaching": "^6.1.0",
     "workbox-routing": "^6.1.0",
```

Comment:
update internal dependencies in according sub package, example: 
add @teleswap/smart-order-route in @teleswap/ui pakcage.json as 
"import { AlphaRouter, AlphaRouterConfig, ChainId } from '@teleswap/smart-order-router'" in packages/ui/src/lib/hooks/routing/clientSideSmartOrderRouter.ts